File: app/src/main/java/me/ycdev/android/devtools/sampler/StatFileLine.java
package me.ycdev.android.devtools.sampler;

import android.os.Environment;

import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.FileWriter;
import java.io.IOException;

import me.ycdev.android.devtools.utils.Constants;
import me.ycdev.androidlib.utils.DateTimeUtils;

public class StatFileLine {
    public long timeUsage;
    public long cpuTime;
    public int processCount;
    public int memPss;
    public int memPrivate;
    public long trafficRecv;
    public long trafficSend;
}

File: app/src/main/java/me/ycdev/android/devtools/sampler/StatFileLine.kt
package me.ycdev.android.devtools.sampler

class StatFileLine {
    var sysTimeStamp: String? = null
    var timeUsage: Long = 0
    var cpuTime: Long = 0
    var processCount = 0
    var memPss = 0
    var memPrivate = 0
    var trafficRecv: Long = 0
    var trafficSend: Long = 0
}
